docs/documentation/cn/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -8,8 +8,9 @@
         [key_desc]
         [partition_desc]
         [distribution_desc]
-        [PROPERTIES ("key"="value", ...)];
-        [BROKER PROPERTIES ("key"="value", ...)];
+        [PROPERTIES ("key"="value", ...)]
+        [BROKER PROPERTIES ("key"="value", ...)]
+        [COMMENT "table comment"];
```

Comment:
To comply with mysql, comment should locate before partition define. Both Comment and Engine are table options. [Here](https://dev.mysql.com/doc/refman/8.0/en/create-table.html) is the reference.

Reply:
Changed